engine/lib/configuration.php

```diff
@@ -275,6 +275,33 @@ function _elgg_load_site_config() {
 }
 
 /**
+ * Set up CONFIG->cookies. (this is for unit testing)
+ *
+ * @see phpunit/bootstrap.php
+ *
+ * @param stdClass $CONFIG Elgg's config object
+ * @access private
+ */
+function _elgg_configure_cookies($CONFIG) {
+	// set cookie values for session and remember me
+	if (!isset($CONFIG->cookies)) {
```

Comment:
Instead of side effects on config, perhaps we could return the cookies array? Then the caller can be responsible for merging the result into `$CONFIG` i.e.: `$CONFIG->cookies = _elgg_configure_cookies();`